src/app/mod.rs

```diff
@@ -607,8 +607,10 @@ impl App {
     }
 
     #[instrument(name = "app.list_wallets", skip_all, err)]
-    pub async fn list_wallets(&self, profile: Profile) -> Result<Vec<Wallet>, BriaError> {
-        self.wallets.list_by_account_id(profile.account_id).await
+    pub async fn list_wallets(&self, profile: Profile) -> Result<Vec<Wallet>, ApplicationError> {
+        println!("{:?}", profile.account_id);
```

Comment:
??

Reply:
I was trying to test something there. Sorry. I will remove it.